# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/base/BrowsingContext.h
# Commit: ab54092b8afc
# Full Hash: ab54092b8afc8e56dd3f8162344a4ce38ec45999
# Author: Niklas Goegge <ngogge@mozilla.com>
# Date: 2021-04-29 09:26:05
# Regressor Bug: 1708150
# File Overlap Count: 1
# Description:
#   Bug 1708150 - Add user activation flag to reload, goBack and goForward r=ckerschb,Gijs,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D110245
# ==============================================================================

diff -r a9a52f01de13 -r ab54092b8afc docshell/base/BrowsingContext.h
--- a/docshell/base/BrowsingContext.h	Wed Apr 28 11:09:38 2021 +0000
+++ b/docshell/base/BrowsingContext.h	Wed Apr 28 11:26:49 2021 +0000
@@ -809,7 +809,7 @@
   GetTriggeringAndInheritPrincipalsForCurrentLoad();
 
   void HistoryGo(int32_t aOffset, uint64_t aHistoryEpoch,
-                 bool aRequireUserInteraction,
+                 bool aRequireUserInteraction, bool aUserActivation,
                  std::function<void(int32_t&&)>&& aResolver);
 
   bool ShouldUpdateSessionHistory(uint32_t aLoadType);